engine/src/main/java/org/pentaho/di/trans/steps/exceloutput/ExcelOutput.java

```diff
@@ -525,8 +526,7 @@ public boolean openNewFile() {
           }
         }
       } else {
-        FileObject templateFile
-            = KettleVFS.getFileObject( environmentSubstitute( meta.getTemplateFileName() ), getTransMeta() );
+        FileObject templateFile = KettleVFS.getFileObject( environmentSubstitute( meta.getTemplateFileName() ), getTransMeta() );
```

Comment:
let's wrap it with try-with-resources to make sure it's closed always, not only in else block below